requirements.in

```diff
@@ -11,7 +11,7 @@ flask
 flask_cors
 python-json-logger
 redis
-rq
+rq<2
```

Comment:
I noticed this was a problem when trying to get a local version of PCG running and so i added it here pre-emptively to save us time in the future when we try to recompile reqs.

Reply:
I think this is fine for now, version 2 was released only recently in Oct.